Drivers/Src/Display.c

```diff
@@ -94,7 +98,7 @@ DisplayError_t Display_Reset(){
 	return Display_Send(restCmd);
 }
 
-DisplayError_t Display_Fault(os_error_loc_t osErrCode, fault_bitmap_t faultCode){
+DisplayError_t Display_Error(os_error_loc_t osErrCode, uint8_t faultCode){
```

Comment:
error_code_t

Reply:
Actually, I realize now that Display_Error() is incompatible with error_code_t because it takes in a uint8_t, but error_code_t is an int16_t. Would it be better to change Display_Error to accept int16_t error codes or standardize the error enums to use uint8_t values?

Reply:
Ok sounds good, then I'll try changing Display_Error to accept a uint16_t as an error code